tests/e2e/test_query_endpoint.py

```
@@ -45,7 +45,7 @@ def test_invalid_question():
         # at least acknowledges that query is non-ocp.
         # Below assert is minimal due to model randomness.
         assert re.search(
-            r"(sorry|questions|assist|can('t|not) help)",
+            r"(sorry|questions|assist|can([’']t|not) help)",
```

Ruff (0.16.1)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)